styles.css

```diff
@@ -0,0 +1,131 @@
+@import url('https://fonts.googleapis.com/css2?family=Poppins:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;0,800;0,900;1,100;1,200;1,300;1,400;1,500;1,600;1,700;1,800;1,900&display=swap');
+
+* {
+  margin: 0;
+  padding: 0;
+}
+
+html {
+  margin: 0;
+  padding: 0;
+  width: 100%;
+}
+
+.header {
+  width: 375px;
+  height: 92px;
+  display: flex;
+  align-items: center;
+  background-color: #3c3a39;
+}
+
+.logo {
+  width: 77px;
+  height: 24px;
+  font-family: Roboto, sans-serif;
+  font-weight: 700;
+  font-size: 18px;
+  line-height: 24px;
+  font-style: bold;
+  align-items: center;
+  color: #fff;
+}
+
+.navbar {
+  left: 24px;
+  display: flex;
+}
+
+.top-section {
+  background-color: #1c1a19;
+  width: 375px;
+  height: 600px;
+  display: flex;
+  align-items: center;
+  background-image: url(./images/bgimage.png);
+  background-repeat: no-repeat;
+  background-position: 100%;
+  background-position-x: bottom-right;
+}
+
+section {
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  height: 100%;
+}
+
+.title {
+  width: 372px;
+  height: 124px;
+  margin-top: 120px;
+  margin-left: 24px;
+  display: flex;
+  font-family: Crete Round, sans-serif;
+  font-weight: 400;
+  font-size: 56px;
+  line-height: 64px;
+  letter-spacing: 0.37px;
+  color: #ff6b00;
+}
+
+h3 {
+  color: #fff5e1;
+  width: 327px;
+  height: 27px;
+  left: 24px;
+  margin-top: 20px;
+  margin-left: 24px;
+  font-family: Roboto, sans-serif;
```

Comment:
- [ ] You must wrap `Roboto` in a quote before it could be detected. Like this 'Roboto'.